sub/img_convert.c

```diff
@@ -31,7 +31,8 @@
 void mp_blur_rgba_sub_bitmap(struct sub_bitmap *d, double gblur)
 {
     struct mp_image *tmp1 = mp_image_alloc(IMGFMT_BGRA, d->w, d->h);
-    if (tmp1) { // on OOM, skip region
+    MP_HANDLE_OOM(tmp1);
+    {
```

Comment:
I'd remove the block. Seems unnecessary after the change.

Reply:
It is indeed. My idea was to avoid unnecessary changes by not un-indenting.